gcloud/bigtable/happybase/batch.py

```diff
@@ -133,3 +191,54 @@ def __exit__(self, exc_type, exc_value, traceback):
         # NOTE: For non-transactional batches, this will even commit mutations
```

Comment:
:(

Is this behavior considered surprising? If so, I feel it should be documented somewhere that's visible.